src/transformers/models/chameleon/modeling_chameleon.py

```diff
@@ -80,7 +80,8 @@ def forward(self, hidden_states):
 ALL_LAYERNORM_LAYERS.append(ChameleonRMSNorm)
 
 
-# Copied from transformers.models.llama.modeling_llama.LlamaRotaryEmbedding with Llama->Chameleon
+# copied from transformers.models.llama.modeling_llama.LlamaRotaryEmbedding with Llama->Chameleon
+# TODO(joao): add me back asap :)
```

Comment:
https://github.com/huggingface/transformers/pull/31999, which propagates the changes to all models, will fix this.